Makefile

```diff
@@ -14,6 +16,14 @@ lint: get_deps
 test: get_deps
 	yarn test
 
+test_integration:
+	- ${DOCKER_COMPOSE_TEST} down
+	${DOCKER_COMPOSE_TEST} up -d
+	./.scripts/docker/wait-healthy.sh test_rabbitmq 20
+	./.scripts/docker/wait-healthy.sh test_reviewer_mocks 60
+	./.scripts/docker/wait-healthy.sh test_continuum_auth 60
+	CONFIG_PATH=./tests/config/continuum-auth.json yarn test:integration
+	
```

Comment:
I think we should also bring down the containers afterwards as we do in the audit integration tests:
```
	- ${DOCKER_COMPOSE_TEST} down
```